src/main/scala/com/databricks/labs/overwatch/env/Database.scala

```diff
@@ -129,6 +133,7 @@ class Database(config: Config) extends SparkSessionWrapper {
     var finalDF: DataFrame = df
     finalDF = if (target.withCreateDate) finalDF.withColumn("Pipeline_SnapTS", pipelineSnapTime) else finalDF
     finalDF = if (target.withOverwatchRunID) finalDF.withColumn("Overwatch_RunID", lit(config.runID)) else finalDF
+    finalDF = if (target.withWorkspaceFriendlyName) finalDF.withColumn("Workspace_Name", lit(config.workspaceFriendlyName)) else finalDF
```

Comment:
How it will be migrated for existing data?

Reply:
during the upgrade script